linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/tests/correct_value.pass.sh

```diff
@@ -1,17 +1,15 @@
 #!/bin/bash
-# platform = Red Hat Enterprise Linux 7,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_ubuntu
+# platform = Oracle Linux 7,Red Hat Enterprise Linux 7,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ubuntu
 # variables = var_password_pam_retry=3
 
-retry_cnt=3
-{{% if 'ubuntu' not in product %}}
-config_file="/etc/pam.d/system-auth"
-{{% else %}}
-config_file="/etc/pam.d/common-password"
-echo "password requisite pam_pwquality.so" > $config_file
-{{% endif %}}
+source common.sh
 
-if grep -q "pam_pwquality\.so.*retry=" "$config_file" ; then
-	sed -i --follow-symlinks "/pam_pwquality\.so/ s/\(retry *= *\).*/\1$retry_cnt/" $config_file
-else
-	sed -i --follow-symlinks "/pam_pwquality\.so/ s/$/ retry=$retry_cnt/" $config_file
-fi
+for file in ${configuration_files[@]}; do
+{{{ bash_ensure_pam_module_option('/etc/pam.d/$file',
+                                   'password',
+								   'required',
+								   'pam_pwquality.so',
+								   "retry",
+								   "3",
+								   "3") }}}
```

Comment:
This last `3` is not necessary here. It was inherited from the old macro. However, since it is not actually changing anything in the functionality of this test scenario, I am fine to waive it and keep this as an improvement opportunity for another PR.